src/CosmosDB/CosmosDB/help/New-AzCosmosDBAccount.md

```diff
@@ -675,11 +675,11 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## INPUTS
 
-### Microsoft.Azure.Commands.CosmosDB.Models.PSCorsRule[]
+### None
```

Comment:
I never saw "None" before. Does it make sense to you?

Reply:
I can find many cases from our repo https://cs.github.com/Azure/azure-powershell?q=%23%23%23+None. It exists but does not make sense.  It seems no harm from my perspective.